### ISSUE TITLE
[FR] Any plans to allow toggling Juno IDE on and off in Atom?

### ISSUE THREAD

**Issue by gtollefson** [JunoLab/Juno.jl#284]
Does Juno / Atom support temporarily disabling Juno IDE in Atom? 

I use Atom for a variety of work outside of using Julia, as do many other Juno users I'd imagine. Is there an easy way to switch Juno off and back on when writing in Julia?

Here's a related [issue](https://github.com/JunoLab/uber-juno/issues/5) that's been closed that doesn't answer this request.

**Comment by pfitzseb** [comment]
You can disable the `julia-client` package and restart Atom. I don't have any plans to make this easier, but it might eventually come for free while refactoring.
Is there something specific that you're annoyed by when Juno is loaded?

**Comment by slightknack** [comment]
I've run into a similar issue, I'll lay out my thoughts here. 

I've recently gotten into Julia and find Juno's tooling to be quite nice. However, most of the work I do is non-Julia-related. I dislike that Juno 'takes control' of Atom turning it into a Julia IDE, and wish it functioned more as a language server with IDE-like features. In a perfect world, I'd be able to toggle Juno into effect when working on Julia-related code-bases, while still being able to use plain old Atom otherwise. 

> Is there something specific that you're annoyed by when Juno is loaded?
I know that question isn't _specifically_ directed at me but here's my answer. Out of the box, Juno adds a toolbar, automatically lays out windows, and drastically increases Atom's startup time (~3 seconds, up from about 1). Additionally, to disable Juno, I have to find the appropriate packages and restart the editor, which is cumbersome. Currently, to work on a new non-Julia project, I have to: Open the project in Atom, Go to packages, Disable Juno-related ones, Close Atom, and Reopen the project. Ideally I'd be able to: Open the project in Atom and Disable Juno (from the menu or through a shortcut).

TL;DR: Overall, I find Juno to be quite nice—but I use Atom for more than just Julia. I wish Juno acted as a separate toggle-able layer built _on top_ of Atom rather than as an IDE that _takes control_ of it. 

Thank you for all y'all's work on Juno so far—I'm a bit new to Julia/Juno, so if there's anything simple I'm missing or you see ways I can solve any of the small annoyances I've encountered, I'd appreciate it if you'd kindly point them out 😄.
